src/backend/adapters/SqliteUserRepository.ts

```diff
@@ -0,0 +1,109 @@
+import { DateTime } from "luxon"
+import { UserPass } from "../domain/models/User"
+import { UserRepository } from "../services/UserService"
+import { sql } from "../utils/sql"
+import { SqliteConnection } from "../utils/sqliteConnection"
+import { ErrorType } from "../utils/tryCatch"
+
+export const makeSqliteUserRepository = (
+  sqliteConnection: SqliteConnection,
+): UserRepository => {
+  // Internal db safe format for the User. Safe fields for sqlite
+  type DbUserTruely = {
+    id: string
+    name: string
+    email: string
+    birthday: string | null
+    passwordHash: string
+  }
```

Comment:
i'm realizing that I want to reduce inputs to fields in very primitive ways, so i can put values in the db.  values like string, number are ok, but DateTime not ok to serialize into the db.  Also boolean is not the best to put in the db.  you CAN insert boolean into sqlite columns but then when you grab it out it's actually retreived as a 1 | 0.   So best to keep this type closer to the db layer.  And I'll expose a different type elsewhere that's more "app logic friendly"